tools/SchemaManager.Core/SqlSchemaManager.cs

```diff
@@ -267,10 +272,10 @@ private async Task<string> GetScriptAsync(int version, string scriptUri, Cancell
 
         private async Task UpgradeSchemaAsync(int version, string script, CancellationToken cancellationToken)
```

Comment:
Looks like this is specifically for initializing the schema now, can you rename it so that's clear?